composer.json

```diff
@@ -46,7 +35,9 @@
         "behat/mink-selenium-driver": "*",
         "behat/mink-sahi-driver": "*",
         "behat/mink-selenium2-driver": "*",
-        "ezsystems/behatbundle": "@dev"
+        "ezsystems/behatbundle": "@dev",
+        "sensio/generator-bundle": "~2.3",
+        "egulias/listeners-debug-command-bundle": "~1.9"
```

Comment:
aha, good point.
site development != cms development
still, should we use composer dev-mode for all ez installations which are used for site development?